### PR TITLE
fix(docs): Remove Aqua CLI

### PR DESCRIPTION
**Pull request by InversionSpaces**

### CODE REVIEW

File: docs/aqua-book/introduction.md [hunk 1]
```diff
@@ -11,11 +11,4 @@ At the core of Fluence is the open-source language **Aqua** that allows for the
...
 
-This book is dedicated to all things Aqua and currently in its **alpha** version and we expect to expand both Aqua's breadth and depth coverage over the coming weeks.
-
-Stay in touch or contact us via the following channels:
-
-- [Discord](https://discord.gg)
-- [Telegram](https://t.me/fluence_project)
-- [Aqua Github](https://github.com/fluencelabs/aqua)
-- [Youtube](https://www.youtube.com/channel/UC3b5eFyKRFlEMwSJ1BTjpbw)
+This book is dedicated to all things Aqua and currently in its **alpha** version and we expect to expand both Aqua's breadth and depth coverage.
```
Comment (shamsartem) [inline docs/aqua-book/introduction.md]:
```suggestion
This book is dedicated to all things Aqua and currently it's in **alpha** stage and we expect to expand both Aqua's breadth and depth coverage.
```

Comment (InversionSpaces) [reply docs/aqua-book/introduction.md]: I rephrased this sentence

**Review by mikevoronov — DISMISSED** [review]
utACK

File: docs/old-build/tutorials/deploy-a-local-fluence-node.md [hunk 2]
```diff
@@ -208,11 +208,11 @@ func greeting(name:string, node:string, greeting_service_id: string) -> string:
   <- res
 ```
 
-We run the script with [`aqua`](../../aqua-book/aqua-cli/aqua-cli.md)
+We run the script with `fluence`
```
Comment (shamsartem) [inline docs/old-build/tutorials/deploy-a-local-fluence-node.md]:
```suggestion
We run the script using [Fluence CLI](https://github.com/fluencelabs/cli)
```

**Review by alari — APPROVED** [review]

**Review by mikevoronov — APPROVED** [review]
utACK

**Review by shamsartem — APPROVED** [review]
thanks

**Pull request merged**